app/models/planning_application.rb

```diff
@@ -14,9 +14,9 @@ class PlanningApplication < ApplicationRecord
   belongs_to :user, optional: true
   belongs_to :local_authority
 
-  before_create :set_target_date
+  before_create :set_target_date, :set_expiry_date
   before_create :set_change_access_id
-  before_update :set_target_date
+  before_update :set_target_date, :set_expiry_date
 
```

Comment:
would it not be best to just do it all in one method? why have two separate method calls 🤔